src/pushsource/_impl/schema/staged-schema.yaml

```diff
@@ -173,6 +173,18 @@ definitions:
                                 minimum: 0
                                 maximum: 65536
 
+            access_endpoint_url:
+                type:
+                - object
+                - "null"
+                properties:
+                    port:
+                        type: number
+                        minimum: 0
+                        maximum: 65536
+                    protocol:
+                        enum: ["http", "https"]
```

Comment:
Did you confirm the supported types are just these in some docs? Or are we just following the logic here as the endpoint must be a web one?

Reply:
I checked via the WebUI when creating a version.

Reply:
Related to my other comment about "scheme" vs "protocol", does this follow some existing API which uses "protocol"?